ginga/misc/plugins/Contents.py

```diff
@@ -124,6 +128,20 @@ def recreate_toc(self):
                 new_highlight |= channel.extdata.contents_old_highlight
         self.update_highlights(set([]), new_highlight)
 
+        # Resize column widths
+        n_rows = self.nrows()
```

Comment:
How about:
```python
n_rows = sum(map(len, self.name_dict.values()))
```?

This avoids a function call and a `for` iteration--the slowest parts of python